### ISSUE TITLE
mailing list needs update in dsDemos

### ISSUE THREAD

**Issue by kupiqu** [DynaSim/DynaSim#270]

**Comment by jsherfey** [comment]
I agree, but we first need to decide on a single approach to the mailing
list.

Three options:

   1. *Google mailing lists* (pros: archives the emails, provides a nice
   interface for browsing and searching through old threads, and makes it easy
   to customize the Welcome message; cons: requires a google account to sign
   up or an administrator to manually invite non-google users)
   2. *Majordomo via BU* (pros: anyone can sign up; cons: old software
that *does
   not archive emails*, and requires users to send emails with majordomo
   commands to manage their subscription/unsubscription/etc)
   3. *Another online service that hosts mailing lists* (Pros:
   customization, feature rich, nice interface; cons: fees that increase with
   the number of users, and some require users to register an account)

Currently, we have been using Google mailing lists, and I've had to
manually invite users without google accounts. I've also created BU
majordomo lists, but their lack of archiving capabilities seems like a
major flaw. BU does not provide any other mailing list services. The only
free online mailing list host I've found is Wiggio, and I would have
suggested we use that, but they are discontinuing their free service in
Nov. Gaggle Mail has good reviews but charges $0.40 per member per year;
Mobilize is also highly rated but charges $0.09 per member per year.

What do you think is the best way to proceed with the mailing list?

**Comment by kupiqu** [comment]
I see

**Comment by kupiqu** [comment]
Jason, I have been using bu mailing list recently (2-3 emails sent), wondering whether anyone saw them and if I should stop using that list then...

**Comment by jsherfey** [comment]
I received a notification that you signed up but did not receive any of your subsequent emails. BU IT seems not to provide any support beyond setting up the list. Does anyone know how to troubleshoot a majordomo mailing list?

I think we should use the Google mailing list for now since it is working for us, but it would be better if we found an alternative that provided archiving and enabled anyone to subscribe.
